pkgs/applications/networking/remote/x2gokdriveclient/default.nix

```
@@ -0,0 +1,66 @@
+{
+  stdenv,
+  lib,
+  fetchurl,
+  qt5,
+  qtbase,
+  qtx11extras,
+  qttools,
+  zlib,
+  gnumake,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "x2gokdriveclient";
+  version = "0.0.0.1";
+
+  src = fetchurl {
+    url = "https://code.x2go.org/releases/source/${pname}/${pname}-${version}.tar.gz";
```

```suggestion
    url = "https://code.x2go.org/releases/source/x2gokdriveclient/x2gokdriveclient-${version}.tar.gz";
```
we no longer do that because when converting to finalAttrs this is almost most likely wrong.